static/app/components/replays/walker/chevronDividedList.tsx

```diff
@@ -36,9 +36,11 @@ const List = styled('ul')<{cols: number}>`
   display: grid;
   gap: ${space(1)};
   grid-template-columns: ${p =>
-    `minmax(auto, max-content) repeat(${
-      (p.cols - 2) * 2 + 1
-    }, max-content) minmax(auto, max-content)`};
+    p.cols <= 3
+      ? `minmax(auto, max-content) max-content minmax(auto, max-content) max-content minmax(auto, max-content)`
+      : `minmax(auto, max-content) repeat(${
+          (p.cols - 2) * 2 + 1
+        }, max-content) minmax(auto, max-content)`};
```

Comment:
in the <=3 case we're defining 5 columns: `$content divider $content divider $content` and we will use either 1, 3, or all of those columns to render

In the previous case, we are more specific, it's always 5 columns, but the middle isn't going to truncate.